src/main/java/com/icthh/xm/ms/timeline/config/tenant/TenantManagerConfiguration.java

```diff
@@ -98,4 +115,23 @@ public TenantProvisioner loggerTenantProvisioner() {
     private String readResource(String location) {
         return IOUtils.toString(new ClassPathResource(location).getInputStream(), UTF_8);
     }
+
+    @SneakyThrows
+    private String getTopicContent() {
+        String json = readResource(Constants.TOPIC_CONFIG_PATH);
+        TopicConsumersSpec topicConsumersSpec = mapper.readValue(json, TopicConsumersSpec.class);
+        Assert.notNull(topicConsumersSpec,
```

Comment:
propose to use Objects.requirenonNull() from JDK

Reply:
fixed